redblueapp/redchallenge3lambda/template.yaml

```diff
@@ -0,0 +1,36 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: Validates code for red team challenge 3.
+
+Globals:
+  Function:
+    Timeout: 3
+
+Resources:
+  ValidateCodeFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: ValidateCodeFunction
+      CodeUri: ./
+      Handler: app.challengeValidator
+      Runtime: nodejs12.x
+      Environment:
+        Variables:
+          CHALLENGE_MASTER_SALT: 121212
+      Events:
+        ValidateCode:
+          Type: Api
+          Properties:
+            Path: /validate-code
+            Method: post
+
+Outputs:
+  ValidateCodeApi:
+    Description: "API Gateway endpoint URL for Prod stage for Validate Code function"
```

Comment:
We don't need an API gateway url. The lambda should be accessed by ARN using the AWS CLI